tests/components/schlage/test_binary_sensor.py

```diff
@@ -31,9 +34,20 @@ async def test_keypad_disabled_binary_sensor(
 
     mock_lock.keypad_disabled.assert_called_once_with([])
 
+    mock_schlage.locks.return_value = []
+    # Make the coordinator refresh data.
+    async_fire_time_changed(hass, utcnow() + timedelta(seconds=31))
```

Comment:
Please use the freezer
```py
freezer.tick(timedelta(seconds=30))
async_fire_time_changed(hass)
```
(I am assuming the 1 second is to go after the update, but we can do with exactly the update moment)

Reply:
👍🏻 done.